gtkm/auth/service_auth.py

```diff
@@ -42,7 +39,9 @@ async def logout(response: Response, session_id: uuid.UUID = Depends(cookie)):
 @auth.get("/api/auth/github/authorize")
 async def authorize_github():
     response = RedirectResponse(
-        url=f"http://github.com/login/oauth/authorize?client_id={CLIENT_ID}")
+        url=
+        f"http://github.com/login/oauth/authorize?client_id={os.environ.get('CLIENT_ID')}"
+    )
     return response
```

Comment:
Czym jest podyktowana ta zmiana?

Reply:
Podając plik .env poprzez argumenty kwargs przy odpalaniu procesu serwisu, jest on zaczytywany po inicjalizacji serwisu, przez co credentiale były None i nie była możliwa autoryzacja. Może i da się to zrobić inaczej, ale to był najszybszy sposób, żeby znów działało.